.github/DEVELOPMENT.md

```diff
@@ -32,18 +33,34 @@ Install the .NET MAUI workload using the versions from a particular branch:
 
 #### Preview 14 branch
 
-```
+
+Windows:
+
+```bat
 iwr https://aka.ms/dotnet/maui/maui-install.ps1 -OutFile maui-install.ps1;
 .\maui-install.ps1 -b 'release/6.0.2xx-preview14' -v '6.0.200-preview'
-``` 
+```
+
+MacOS:
+
+```bash
+curl -O -L https://aka.ms/dotnet/maui/maui-install.ps1
+chmod +x maui-install.ps1
+sudo pwsh
+./maui-install.ps1 -b 'release/6.0.2xx-preview14' -v '6.0.200-preview'
```

Comment:
What does this look like if you do `sudo pwsh -File`, can you remove the `chmod` in that case?

Maybe we can drop two commands in this example?

Reply:
Ah nice catch! Updated 😄